PyViCare/PyViCareDevice.py

```diff
@@ -15,3 +15,19 @@ def __init__(self, service: ViCareService) -> None:
     @handleNotSupported
     def getSerial(self):
         return self.service.getProperty("device.serial")["properties"]["value"]["value"]
+
+    @handleNotSupported
+    def isHeatingDevice(self):
```

Comment:
What do you think of renaming this function in something like `isDeviceHeatingEnabled` ? Because I don't know which impact on this data point is made by changing the active mode of the Heating Device (e.g. DHW only during Summer)
I think the same renaming could be useful for the other functions as well.

Reply:
I can test it later with my Oil Boiler and Heatbox2

Reply:
There is also a status

<img width="242" alt="Bildschirmfoto 2024-04-03 um 11 13 50" src="https://github.com/somm15/PyViCare/assets/9592452/e629571c-a911-4d75-bc24-1db10dcd2e59">

Reply:
Can you also verify if this influences the ["properties"]["active"]["value"] of DHW? 
So does turning of the DHW (is that possible?) disable the flag?

Reply:
If I disable DHW on my boiler, the feature `heating.dhw` changes `isEnabled` to false and removes the properties
`{
            "feature": "heating.dhw",
            "gatewayId": "xxxx",
            "deviceId": "0",
            "timestamp": "2024-04-04T16:01:14.844Z",
            "isEnabled": false,
            "isReady": true,
            "apiVersion": 1,
            "uri": "https://api.viessmann.com/iot/v1/features/installations/xxx/gateways/xxxxxx/devices/0/features/heating.dhw",
            "properties": {},
            "commands": {}
        }`
To disable dhw I need to reconfigure it and remove the dhw temp. Sensor
So my Boiler acts as a device where DHW is possible but not enabled